packages/components/src/internal/components/samples/SampleDetailEditing.tsx

```diff
@@ -27,9 +32,11 @@ interface Props {
     detailRenderer?: DetailRenderer;
     onEditToggle?: (isEditing: boolean) => void;
     onUpdate: () => void;
-    queryModel?: QueryModel;
     sampleSet: string;
     title: string;
+    queryModel?: QueryModel;
+    isAliquot?: boolean;
+    aliquotRootLsid?: string;
```

Comment:
I'm confused by these two new properties. It seems like the changes below in this component are instead calculating them based on the sample row data. If that is true, can these be removed as props?